adapters/controllers/film_controller.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"wizeline/ghibli/adapters/repository"
+)
```

Comment:
For legibility, I would recommend you to separate the std imports from your own packages like this:

```suggestion
import (
	"encoding/json"
	"net/http"
	"strconv"
	
	"wizeline/ghibli/adapters/repository"
)
```

As well as adding the full repository to your package import.

Reply:
Done. I think using the full URL as a module name is rather ugly, but it seems to be the norm.